src/l500/l500-options.cpp

```diff
@@ -64,27 +93,50 @@ namespace librealsense
         auto min_value = float(*(reinterpret_cast<int32_t*>(min.data())));
 
         _range = option_range{ min_value,
-            max_value,
-            float(*(reinterpret_cast<int32_t*>(step.data()))),
-            def };
+                               max_value,
+                               float( *( reinterpret_cast< int32_t * >( step.data() ) ) ),
+                               query_default() };
+    }
+
+    void l500_hw_options::update_default()
+    {
+        _range.def = query_default();
     }
 
     void l500_hw_options::enable_recording(std::function<void(const option&)> recording_action)
     {}
 
-    float l500_hw_options::query(int mode) const
+    float l500_hw_options::query( l500_command type, int mode ) const
```

Comment:
It's an `op`, not a `type` (`type` is confusing because there's `_type` which is really the control...)

Reply:
done

Reply:
But I don't see what you changed...